tests/integration/kubernetes/gha-run.sh

```diff
@@ -191,7 +191,7 @@ function deploy_kata() {
 }
 
 function install_kbs_client() {
-	echo "TODO: install kbs-client - https://github.com/kata-containers/kata-containers/pull/9114"
+	kbs_install_cli
```

Comment:
More planned to be added here? or is think function redundant?

Reply:
For the time being it is a redundant function, yes. We had plan to add some checkers here for when to skip it, not sure the plan stands.